src/style.css

```diff
@@ -0,0 +1,110 @@
+@font-face {
+  font-family: "Avenir", sans-serif;
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "AvenirHeavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
```

Comment:
Move this property to html, only `box-sizing` allowed for `*` selector

Reply:
Done